droidlet/lowlevel/hello_robot/remote/utils.py

```diff
@@ -238,20 +223,12 @@ def goto(
                 is_obstacle = obstacle_fn()
             if is_obstacle:
                 # stop motion
-                robot.base.set_rotational_velocity(v_r=0.0)
-                robot.push_command()
+                print("Found obstacle while translating. Aborting")
+                robot.stop()
                 return "FAILED"
             time.sleep(0.1)
             robot.pull_status()
-            left_wheel_moving = (
-                robot.base.left_wheel.status["is_moving_filtered"]
-                or robot.base.left_wheel.status["is_moving"]
-            )
-            right_wheel_moving = (
-                robot.base.right_wheel.status["is_moving_filtered"]
-                or robot.base.right_wheel.status["is_moving"]
-            )
-            is_moving = left_wheel_moving or right_wheel_moving
+            is_moving = robot.is_base_moving()
 
     # second rotate by theta2
     theta_2 = np.sign(theta_2) * (abs(theta_2) % radians(360))
```

Comment:
what are theta_1 vs theta_2?

Reply:
they're described in the comments in the file.